src/main/java/net/william278/velocitab/packet/UpdateTeamsPacket.java

```diff
@@ -86,11 +115,60 @@ protected static UpdateTeamsPacket removeFromTeam(@NotNull String teamName, @Not
                 .entities(Arrays.asList(teamMembers));
     }
 
+    @NotNull
+    protected static UpdateTeamsPacket removeTeam(@NotNull String teamName) {
+        return new UpdateTeamsPacket()
+                .teamName(teamName.length() > 16 ? teamName.substring(0, 16) : teamName)
+                .mode(UpdateMode.REMOVE_TEAM);
+    }
+
     @NotNull
     private static String getChatString(@NotNull String string) {
         return "{\"text\":\"" + StringEscapeUtils.escapeJson(string) + "\"}";
     }
 
+    public static int getLastColor(@Nullable String text) {
+        if (text == null) {
+            return 15;
+        }
+        int intvar = text.lastIndexOf("§");
+
+        if(intvar == -1 || intvar == text.length() - 1) {
+            return 15;
+        }
+
+        String last = text.substring(intvar, intvar + 2);
+        return convertColorCharToInt(last.charAt(1));
+    }
+
+    public static int convertColorCharToInt(char colorChar) {
+        return switch (colorChar) {
+            case '0' -> 0;
+            case '1' -> 1;
+            case '2' -> 2;
+            case '3' -> 3;
+            case '4' -> 4;
+            case '5' -> 5;
+            case '6' -> 6;
+            case '7' -> 7;
+            case '8' -> 8;
+            case '9' -> 9;
+            case 'a' -> 10;
+            case 'b' -> 11;
+            case 'c' -> 12;
+            case 'd' -> 13;
+            case 'e' -> 14;
+            case 'f' -> 15;
+            case 'k' -> 16; // Obfuscated
+            case 'l' -> 17; // Bold
+            case 'm' -> 18; // Strikethrough
+            case 'n' -> 19; // Underlined
+            case 'o' -> 20; // Italic
+            case 'r' -> 21; // Reset
+            default -> 15; // Invalid
+        };
+    }
```

Comment:
```suggestion
    public enum TeamColor {
        BLACK('0', 0),
        DARK_BLUE('1', 1),
        DARK_GREEN('2', 2),
        DARK_AQUA('3', 3),
        DARK_RED('4', 4),
        DARK_PURPLE('5', 5),
        GOLD('6', 6),
        GRAY('7', 7),
        DARK_GRAY('8', 8),
        BLUE('9', 9),
        GREEN('a', 10),
        AQUA('b', 11),
        RED('c', 12),
        LIGHT_PURPLE('d', 13),
        YELLOW('e', 14),
        WHITE('f', 15),
        OBFUSCATED('k', 16),
        BOLD('l', 17),
        STRIKETHROUGH('m', 18),
        UNDERLINED('n', 19),
        ITALIC('o', 20),
        RESET('r', 21);

        private final char colorChar;
        private final int id;

        TeamColor(char colorChar, int id) {
            this.colorChar= colorChar;
            this.id = id;
        }
        
        @NotNull
        public static int getColorId(char char) {
            return Arrays.stream(values()).filter(color -> color.colorChar == char).findFirst().orElse(15);
        }
    }
```
I think an enumeration approach like this would be a bit more readable here